server/routes/auth/index.js

```diff
@@ -26,6 +26,9 @@ router.post("/register", async (req, res, next) => {
       process.env.SESSION_SECRET,
       { expiresIn: 86400 }
     );
+
+    res.cookie("token", token, { httpOnly: true });
+
     res.json({
       ...user.dataValues,
       token,
```

Comment:
You don't need to return the token in the response since you have it set in the cookie.

Reply:
Thanks! I will remember that next time